api/v0/ingest/schema/schema.ipldsch

```diff
@@ -18,7 +18,15 @@ type Advertisement struct {
     Addresses [String]
     # Signature is the signature of this advertisement.
     Signature Bytes
-    # Entries is a link to the chained EntryChunk instances that contain the multihashes advertised. 
+    # Entries is a link to a data structure that contains the advertised multihashes.
```

Comment:
Update also in https://github.com/filecoin-project/storetheindex/blob/main/doc/ingest.md ?